frontend/src/bundles/chats/helpers/filter-chats.helper.ts

```diff
@@ -0,0 +1,20 @@
+import { type ChatPreviewResponseDto } from '../types/types.js';
+
+const filterChats = (
+    chats: ChatPreviewResponseDto[],
+    search: string,
+): ChatPreviewResponseDto[] => {
+    return chats.filter((chat) =>
+        chat.users.some(
+            ({ fullName, email }) =>
+                (fullName &&
+                    fullName
+                        .toLowerCase()
+                        .includes(search.trim().toLowerCase())) ||
+                (email &&
+                    email.toLowerCase().includes(search.trim().toLowerCase())),
```

Comment:
can we add additional condition to filter users only by fullName if they have it ? because it could be confusing since you will find chats, but chat name will not contain search string